runner/bin/run/check.js

```diff
@@ -6,22 +7,26 @@ const FORM_PATH = path.join(process.cwd(), "src", "server", "forms");
 
 async function check() {
   cli.action.start("Checking versions of forms in runner/src/forms");
-  const files = await fs.readdir(FORM_PATH);
-  const forms = (await files).filter((file) => path.extname(file) === ".json");
+  const files = (await fs.readdir(FORM_PATH)).filter(
+    (file) => path.extname(file) === ".json"
+  );
+  let needsMigration = [];
 
-  const needsMigration = forms.filter(async (file) => {
+  for (const file of files) {
     const form = await fs.readFile(path.join(FORM_PATH, file));
     const version = JSON.parse(form).version || 0;
-    return version < CURRENT_SCHEMA_VERSION;
-  });
+    version < CURRENT_SCHEMA_VERSION && needsMigration.push(file);
+  }
+
   cli.action.stop();
 
   if (needsMigration.length > 0) {
     cli.warn(
```

Comment:
Would a combination of warn  & process.exit be enough here to fail the build

Reply:
yep - in the PR message
failing build is here, before I updated the JSON https://github.com/XGovFormBuilder/digital-form-builder/runs/2139461929?check_suite_focus=true

and then this is after I changed the JSON
pass: https://github.com/XGovFormBuilder/digital-form-builder/pull/444/checks?check_run_id=2140986342